### PR TITLE
docs : Calibration guide

### LINKED ISSUE #865: Add Calibration guide to AirQo docs
**Describe the solution you'd like**
We would like to add the [AirQo Calibration guide](https://docs.google.com/document/d/1z65wAjDscZvA9i_oLyHaNm7xKjkaq_DFBd3vjapVzXA/edit?usp=sharing) to the [AirQo docs](https://staging-docs.airqo.net/#/)

**Additional context**
The [AirQo docs](https://staging-docs.airqo.net/#/) source files can be found [here](https://github.com/airqo-platform/AirQo-frontend/tree/staging/docs)

### PR DESCRIPTION
**Pull request by HassanBahati**
#### Summary of Changes (What does this PR do?)
- This pr adds calibration guide to AirQo docs and addresses #865. 

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* The Calibration Guide docs can be found under the Platform section of the docs.

#### What are the relevant tickets?
- Closes #865 

#### Screenshots (optional)

Screenshot from 2022-10-19 11-49-36
Screenshot from 2022-10-19 11-49-48

### CODE REVIEW

**Comment by HassanBahati** [comment]
@Codebmk , how can the images in [AirQo Calibration guide](https://docs.google.com/document/d/1z65wAjDscZvA9i_oLyHaNm7xKjkaq_DFBd3vjapVzXA/edit?usp=sharing) be accessed so that they be added ?

**Comment by Codebmk** [comment]
Thanks, @HassanBahati . Is this ready for review? Kindly add screenshots of what you have done. Thanks.

File: docs/platform/calibration_guide.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# Field Calibration Guide
+
+
+# Contents
```
Comment (Codebmk) [inline docs/platform/calibration_guide.md:4]: @HassanBahati Could you add links to the items in the content (table of content) section?

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @HassanBahati 
Looks good to me

File: docs/platform/calibration_guide.md (hunk 1 above)
Comment (HassanBahati) [reply docs/platform/calibration_guide.md:4]: @Codebmk i have added the links to the items in the table of content

**Review by Codebmk — APPROVED** [review]
LGTM. Thanks @HassanBahati

**Comment by 123MwanjeMike** [comment]
Hi @HassanBahati 
Thanks again for the great work.
I've just made a bit of file restructure and will add the corresponding images

**Comment by HassanBahati** [comment]
> 

Okay please @123MwanjeMike .

@Codebmk @123MwanjeMike the pleasure is mine. Glad to contribute...

**Comment by 123MwanjeMike** [comment]
image

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @HassanBahati , this is awesome! :)

**Pull request merged**